tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1258,7 +1258,7 @@ def test_generate_with_prompt_ids_and_task_and_language(self):
         config, input_dict = self.model_tester.prepare_config_and_inputs_for_common()
         model = WhisperForConditionalGeneration(config).eval().to(torch_device)
         input_features = input_dict["input_features"]
-        prompt_ids = np.arange(5)
```

Comment:
Previously we allowed passing numpy arrays into `model.generate(...)` which I think was not intentional and a bug (see here: https://github.com/huggingface/transformers/pull/22496/files#r1467370182). 

If people feel strongly I can make it backward compatible though inside generate, but we should definitely not allow passing numpy arrays into a nn.Module

Reply:
Good with me - agree that this is a "bug" that we can fix